### ISSUE TITLE
Nomad+ feedback

### ISSUE THREAD

**Issue by cofauver** [laurakathleen/vagabond-project#16]
### Group: SLAM - Sam, Laura, Ashley, Wolf

_Scoring Key_
- 0 - "Does Not Meet Expectations"
- 1 - "Meets Expectations"
- 2 - "Exceeds Expectations"
### _Project Workflow - 1_
- Frequent Commits with good commit messages: **2**
- Collaboration (cooperative group work with majority pair programing): **1**
- Effective use of branches: **0**
- Deliberate Approach (Routeside-in, logical progress from skateboard to car): **1**
- Original Code: **1**

I think my biggest comment here is about creating a better relationship with git branches. It seems like the branches really became a tangle in this project. You abandoned `master` after 3 days on the project and it seems like you returned to having a `development` branch quite a bit later. Maybe I'm reading the graph wrong in guessing that, but overall, the graph looks pretty chaotic.

![image](https://cloud.githubusercontent.com/assets/6520345/19406604/001f36d8-923e-11e6-9753-1c1d4978cd81.png)

Work to keep a `master` branch that you merge into periodically. These merges can intentionally come only a few times at big landmarks, but you want one main branch (perhaps `development`) that keeps a running compilation of the work being done. Make a feature branch off of that main branch, build a small feature (1-10 commits), and then merge it back into that main branch when it's working. This is more likely to keep everyone on track with the work others are doing. Ideally, it would look more like this:

![image](https://cloud.githubusercontent.com/assets/6520345/19406667/e6c48c00-923e-11e6-9444-39c344a8574a.png)

Outside of the branching, you all seemed to pair program a good amount, although it seemed that some paired more than others. 233 commits! OMGz! Great diligence to committing early and committing often!
### _Execution - 1_
- Fulfills user stories: **1**
- Code is visually clean (Consistent use of indentation, whitespace, no extraneous commented out code): **1**
- Follows good naming conventions (Variable names are semantically clear / correctly cased): **1**
- Code is free of obvious errors/bugs: **1**
- Code demonstrates good problem solving and effective use of Rails: **1**
- Code is DRY: **1**

On the deployed version, the new post form seems to be broken. When I sign up for an account, it doesn't also sign me in. These come slightly short of the user stories.
### _Technical Requirements - 1.2_
- Users authenticated (Fully integrated User model / pages for signup and login): **1**
- Code works (and will work in production): **1**
- Full CRUD for cities: **3/5** **.5**
- Custom JS, HTML, and styling: **2**
- User Authorization (Important endpoints are protected. Can any user edit/delete a post? Access another user's profile?): **1**
- Deployment: **1**
- Comments model (sprint 4 bonus feature): **2**

I don't see the ability to create, delete, or update a city, even in the admin account. I can go to the `/new` endpoint to get to the new city form, but it seems like a button would be good.

I can create a post and delete a post, but when I press the save button to edit it, no update seems to happen.

Good work on getting far with creating commenting!
### _Creativity - 2_
- Visually polished: **2**
- Personal spin: **2**

This is an extremely good looking website! We sometimes hear from employers that student portfolios have good projects that don't look sharp and therefore aren't very appealing. This project looks and feels slick as you navigate through it! Great focus on a ton of the details.

A few details to give feedback on. I found it strange that the cities index page and the cities show page had very different displays of posts. I'm not sure if this behavior is common or just my computer, but I also see that the city show page has the city name partially blocked by the navbar:

![image](https://cloud.githubusercontent.com/assets/6520345/19413943/f4ec89fa-92f1-11e6-90fd-83799c2a9046.png)

The tiles also sometimes align to the right sort of awkwardly because you have tiles of varying height on the row above. Could you standardize the size of these cards/tiles?
### _Group Success (Did this group work together well?) - 1.5_
- This project demonstrates mastery of class material: **2**
- This group is proactive about reaching out / asking questions / getting help: **1**

Everyone reflected on how fun it was to work in this group. I'm happy that you all felt good about your teamwork, because I enjoyed observing it!
### _Summary - 1.4_

Overall, this is an excellent project! I think that your styling scheme stands out. This is a big enough project that you still have a number of bugs and details to work out in the deployed version. This is natural whenever a codebase grows quickly and doesn't have formal testing built in.
